lib/tapioca/runtime/reflection.rb

```diff
@@ -255,6 +255,35 @@ def child_module_for_parent_with_name(parent, name)
       def method_defined_by_forwardable_module?(method)
         method.source_location&.first == Object.const_source_location(:Forwardable)&.first
       end
+
+      sig { params(symbol: String).returns(String) }
+      def namespace_of(symbol)
+        symbol_arr = symbol.split("::")
+        return "" if symbol_arr.length == 1
+
+        T.must(symbol_arr[0...-1]).join("::")
+      end
+
+      sig { params(symbol: String).returns(T::Boolean) }
+      def contains_alias?(symbol)
```

Comment:
Can we rename this to be:
```suggestion
      def has_aliased_namespace?(name)
```